README.md

```diff
@@ -32,7 +32,7 @@ Control Panel.
 
 This gem extends your solidus application by adding a `POST /api/payment_client_token` endpoint to you application to generate Braintree payment client token. This endpoint requires an authentication token in your request header.
 
-It creates a new `PaymentMethod` class called `Solidus::Gateway::BraintreeGateway`. You can configure this payment method in the admin and add your Braintree public/private keys and merchant id. The admin will render a Braintree dropin container when prompting you to create an order payment.
+It creates a new `PaymentMethod` class called `Solidus::Gateway::BraintreeGateway`. You can configure this payment method in the admin and add your Braintree public/private keys, merchant id, and the proper environment. Please ensure that the environment value is included among the values listed [here](https://github.com/braintree/braintree_ruby/blob/7660ae6f6d228acee8eaedd1830afafbaf91820f/lib/braintree/configuration.rb#L235), otherwise you will encounter unexpected errors. The admin will render a Braintree dropin container when prompting you to create an order payment.
```

Comment:
I think it's better to also add an example on how to set that configuration, WDYT?

Reply:
In order to fix it I set the value through the backend interface, and adding a screenshot of what to change doesn't make much sense to me; also I think that my change is coherent with the Usage section's approach, which is "tell to the installer to edit the backend interface"; if we choose to tell to the developer to set the `environment` value in another way it this section becomes more complex IMHO

Reply:
You have more context than me here so if it makes sense for you I'm 👍